packages/devtools_app/lib/src/theme.dart

```diff
@@ -440,7 +440,10 @@ CurvedAnimation defaultCurvedAnimation(AnimationController parent) =>
     CurvedAnimation(curve: defaultCurve, parent: parent);
 
 Color titleSolidBackgroundColor(ThemeData theme) {
-  return theme.canvasColor.darken(0.2);
+  return theme.colorScheme.isLight
+      // This matches the color for alternating row background colors in tables.
```

Comment:
Can you define a color in the theme and use it in both places to keep these in sync?

Reply:
added color to theme extension